index.js

```diff
@@ -0,0 +1,23 @@
+"use strict"
+
+// The list of rules that can be used in Solhint can be found at:
+// https://protofire.github.io/solhint/docs/rules.html
+
+// The rules below use the recommended rules from solhint with some
+// modifications
+module.exports = {
+  "extends": "solhint:recommended",
```

Comment:
@pdyraga 
I think I found an issue here. 
I used this sharable configuration in one of our project and only the rules listed explicitly below are applied there
(all the rules from `recommended` ruleset were lost, e.g. the rules `quotes` and `no-empty-blocks` which are enabled by default in recommended). 

It looks like extending cannot be used in a sharable configuration like ours.

There are two things we can do:
1) List all the rules we want to use explicitly in this sharable (without extending `recommended`) - the best option in my opinion
2) Extend from both `recommended` and  this sharable config in our projects, like that:
```
{
  "extends": ["solhint:recommended", "keep"]
}
```

I think the first solution is the best one.

Reply:
Yes, agreed, let's go with the first option.